### DISCUSSION TITLE
vue 技术栈的是否可以使用

### DISCUSSION THREAD

**Discussion by SamWeiFK** [umijs/dumi#2043] [Q&A] [answered]
@PeachScript 在issue中看相关的意见merged，是否能提供一个最小demo

**Comment by jeffwcx** [comment, marked as answer]
@SamWeichangyue 要稍等一会儿，目前绝大部分功能已经合并了，但是还有一些需要完善的点，比如apiParser(dumi-vue-meta)，live-demo等。本月内都会陆续完成，请耐心等待v2.3的发布

**Reply by SamWeiFK** [reply]
感谢

**Reply by SamWeiFK** [reply]
@jeffwcx 目前使用dumi 最新版，能否提供vue组件的最小示例

**Reply by jeffwcx** [reply]
可以看这个：[examples/vue](https://github.com/umijs/dumi/tree/master/examples/vue)，等稳定之后，会直接提供项目模板，欢迎你试用，多提建议

**Reply by SamWeiFK** [reply]
thanks

**Comment by littleboyck** [comment]
我参考了example/ vue 集成ant-design-vue 报样式文件找不到

**Reply by jeffwcx** [reply]
https://github.com/jeffwcx/dumi-vue-demo 并没有发现太大问题啊

**Comment by willem-wy** [comment]
https://zhuanlan.zhihu.com/p/2036816407548141632   这个也可以哦，@willem-wy/dumi-plugin-preset-vue2为Dumi 2提供 Vue 2 组件渲染支持，并且实现 Vue 2 与react组件共存。
